ios/FluentUI/Extensions/UIImage+Extensions.swift

```diff
@@ -14,17 +14,6 @@ import UIKit
     }
 
     @objc func image(withPrimaryColor primaryColor: UIColor) -> UIImage {
```

Comment:
we should start deprecating this function from this extension I think.

Reply:
I'll create a separate work item to address that.